common-system.js

```diff
@@ -239,7 +239,7 @@ System.prototype.loadSystemDescription = function loadSystemDescription(location
             throw error;
         }
     }, function (error) {
-        error.message = "Can't load package " + JSON.stringify(name) + " at " +
+        error.message = "Can't load package " + JSON.stringify(location) + " at " +
```

Comment:
This is just generally a previously untested bug, that you get trying `node bundle.js boot-entry.js` on master with node 6.